### PR TITLE
Content: 2 new jobs kinds for the Unfettered, mining and medical rush

### PR DESCRIPTION
**Pull request by Hurleveur**
**Content (Jobs)**

## Summary
This is in the continuing effort to add a bit more content to the Unfettered and cut away from the campaign bit by bit into the game. I took the generic jobs that could be offered from the start to put them here, so we can work out balance etc.

A 2 new kind of generic jobs:
-mining, the Unfettered ask you to bring those minerals back, paying roughly 5x their value, and asking for the minerals in UHai space (silver, silicon and lead)
-rush medicals, they come in small and big versions, and pay you quite a bit, but unlike the tribute these are timed, and not that likely, so you can't really combine them with many things.

Next up there will be some more advanced jobs...

## Save File
Any would work, these are in Unfettered space and accessible from the start

## Artwork Checklist
N/A

### CODE REVIEW

File: data/hai/unfettered jobs.txt [hunk 1]
```diff
@@ -107,3 +108,167 @@ mission "Unfettered Jump Drive 4"
...
+	source
+		attributes "unfettered"
+	stopover
+		distance 5 7
+		government "Hai"
+		attributes "spaceport"
+	destination
+		attributes "unfettered"
+	to offer
+		has "Unfettered: Bring Medical Supplies: done"
```
Comment (Anarchist2) [inline data/hai/unfettered jobs.txt:266]: Is this a real mission?

**Review by Anarchist2 — CHANGES_REQUESTED** [review]

File: data/hai/unfettered jobs.txt [hunk 2]
```diff
@@ -107,3 +108,167 @@ mission "Unfettered Jump Drive 4"
...
+		`Once the unloading of materials is finished, an Unfettered comes to you, looking humble and sincere. "Thank you, captain. We need these materials - and any other kind of help - more than we would like to admit it. If you have time, please consider doing other jobs, such as bringing food for us," he says with a slight smile. He then gives you <payment>.`
+
+mission "Unfettered Silicon Asteroid Mining"
+	job
+	repeat
+	name `Gather silicon`
+	description `Mine and then collect 50 tons of silicon from the asteroid field in Wah Yoot, and then bring them back to <destination> for <payment>.`
+	source "Darkcloak"
+	to offer
+		random < 40
```
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:131]: They could bring in multiple small parts or smth, or do it later
And 40 tons isnt much

[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:131]: Yeah it is this is just basic materials that they buy back from you, as if they were buying food
Tributes require more trust than this

File: data/hai/unfettered jobs.txt (hunk 1 above)
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:267]: Oh crap not meant to be there

File: data/hai/unfettered jobs.txt [hunk 3]
```diff
@@ -107,3 +108,167 @@ mission "Unfettered Jump Drive 4"
...
+	deadline 15
+	cargo "medicals" 35 2 .05
+	source
+		attributes "unfettered"
+	stopover
+		distance 5 7
+		government "Hai"
+		attributes "spaceport"
+	destination
+		attributes "unfettered"
```
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:265]: I dont like that, it makes it messy, all off them pointing to the same world
And the world would call for this on the nearby ones

File: data/hai/unfettered jobs.txt [hunk 4]
```diff
@@ -107,3 +108,167 @@ mission "Unfettered Jump Drive 4"
...
+	to offer
+		has "Unfettered: Bring Medical Supplies: done"
+		random < 40
+	on visit
+		dialog phrase "generic missing stopover or cargo"
+	on stopover
+		dialog phrase "unfettered medicals pickup dialog"
+	on complete
+		payment 8000 3000
```
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:274]: Oh I did not mean to make the deadline stricter but the opposite

File: data/hai/unfettered jobs.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:265]: Also all planets need food, but not all will need meds all the time

File: data/hai/unfettered jobs.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:131]: Maybe the medicals could require a bit more... I imagine they only ask you if they really have to

File: data/hai/unfettered jobs.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Anarchist2) [reply data/hai/unfettered jobs.txt:265]: The vast majority of stopover jobs point back at the source; I don't see a strong argument for not doing that for this mission.

File: data/hai/unfettered jobs.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Anarchist2) [reply data/hai/unfettered jobs.txt:131]: 50 tons is a lot for someone in a single ship, and even more so if they're using outfit expansions. (Knowing to put minerals in storage is also pretty unintuitive.)

File: data/hai/unfettered jobs.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Anarchist2) [reply data/hai/unfettered jobs.txt]: Better, but the payment for both rush missions still feels too high. Doing a little bit of testing, I've found jobs that pay 5.6 million for 101 tons, 2.7 million for 69 tons and 2.8 million for 59 tons.

[the remark this answers is not in the document]
Comment (Anarchist2) [reply data/hai/unfettered jobs.txt]: The missions are pretty lenient, though. On the maximum distance job, you're getting ~24 days to travel 14 jumps.

[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt]: I'm kinda new with these kind of jobs tbh, I made some tweaks to the payments and how likely they are to offer
 I'll make the first mission a bit more stressful then

[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt]: It doesnt count the way back actually, so I'll modify it a bit.
(the second thing in deadline is only the distance until the stopover and without taking the way back into account)

File: data/hai/unfettered jobs.txt [hunk 5]
```diff
@@ -108,3 +110,173 @@ mission "Unfettered Jump Drive 4"
 		outfit "Hyperdrive" 1
 		payment 1000000
 		"reputation: Hai (Unfettered)" >?= 40
+
+
+phrase "unfettered mining payment dialog"
+	word
+		`The group of Unfettered unloading the materials you mined seems enthused. "Keep this up and we will be building new ships in no time! Some use asteroids as target practice, but find it too dull and easy to fight an enemy that does not react," one tells you while handing over your payment of <payment>.`
+		`The local Unfettered unload your cargo of minerals, and one comes you give you your payment of <payment>. "Some Unfettered use these asteroids as target practices, but too few bring the precious materials back," he adds.`
```
Comment (tibetiroka) [inline data/hai/unfettered jobs.txt:118]:
```suggestion
		`The local Unfettered unload your cargo of minerals, and one comes to give you your payment of <payment>. "Some Unfettered use these asteroids as target practices, but too few bring the precious materials back," he adds.`
```

File: data/hai/unfettered jobs.txt [hunk 6]
```diff
@@ -108,3 +110,173 @@ mission "Unfettered Jump Drive 4"
...
+		`A group of Unfettered unload the materials you mined, without saying anything to you. When they are done, one gives you <payment>.`
+		`The leader of the Unfettered unloading the materials comes to you. "This will allow us to repair our houses, our hospitals, even our schools. For that you have my deepest thanks." He then hands you <payment>.`
+		`The local Unfettered collect the minerals and happily hand you <payment>.`
+		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit it." He then gives you <payment>.`
+
+mission "Unfettered Silicon Asteroid Mining"
+	job
+	repeat
+	name "Gather Silicon from asteroids"
+	description "Mine and collect 50 tons of silicon from the asteroid field in Wah Yoot, then bring the minerals to <destination> for <payment>."
```
Comment (tibetiroka) [inline data/hai/unfettered jobs.txt:129]: Does it matter where you get the minerals from? It seems weird to have the system in the description like it was the only option, but as far as I can see this doesn't actually matter (and is just a recommendation / help).

**Review by tibetiroka — CHANGES_REQUESTED** [review]
Quick grammar check.

File: data/hai/unfettered jobs.txt (hunk 6 above)
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:129]: Yeah its just a recommendation
At first I had set a waypoint but then u cant complete it without going there

File: data/hai/unfettered jobs.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:132]: I'll do it for the big mission but I want the player to see these even if they dont have cargo rn

[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:132]: I made it for all missions, if you have a bit under it you'll see them and the smallest one is always offered regardless of cargo so the player knows its smth they can do

[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:132]: I made them be more likely to offer if you've done the JD
But if they really need to they'll ask you anyway

File: data/hai/unfettered jobs.txt [hunk 7]
```diff
@@ -108,3 +110,178 @@ mission "Unfettered Jump Drive 4"
 		outfit "Hyperdrive" 1
 		payment 1000000
 		"reputation: Hai (Unfettered)" >?= 40
+
+
+phrase "unfettered mining payment dialog"
+	word
+		`The group of Unfettered unloading the materials you mined seems enthused. "Keep this up and we will be building new ships in no time! Some use asteroids as target practice, but find it too dull and easy to fight an enemy that does not react," one tells you while handing over your payment of <payment>.`
+		`The local Unfettered unload your cargo of minerals, and one comes to give you your payment of <payment>. "Some Unfettered use these asteroids as target practices, but too few bring the precious materials back," he adds.`
```
Comment (roadrunner56) [inline data/hai/unfettered jobs.txt:118]:
```suggestion
		`The local Unfettered unload your cargo of minerals, and one comes to give you your payment of <payment>. "Some Unfettered use these asteroids as target practice, but too few bring the precious materials back," he says.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by roadrunner56 — APPROVED** [review]
Grammar looks good.

File: data/hai/unfettered jobs.txt [hunk 8]
```diff
@@ -108,3 +110,174 @@ mission "Unfettered Jump Drive 4"
...
+		`The Unfettered unloading the materials you mined seem satisfied. "This will allow us to repair our ships! Too few captains are patient enough to gather asteroids like you are. Most of them try to get the resources from enemy ships, but this is a safer way to do it," one tells you before doling out your payment of <payment>.`
+		`A group of Unfettered unload the materials you mined, without saying anything to you. When they are done, one gives you <payment>.`
+		`The leader of the Unfettered unloading the materials comes to you. "This will allow us to repair our houses, our hospitals, even our schools. For that you have my deepest thanks." He then hands you <payment>.`
+		`The local Unfettered collect the minerals and happily hand you <payment>.`
+		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit it." He then gives you <payment>.`
```
Comment (warp-core) [inline data/hai/unfettered jobs.txt:123]: The first and third lines seem to have very similar structures and could probably be split up into words for more variety in the resulting phrases. Same for the fifth and seventh lines.

Comment (warp-core) [inline data/hai/unfettered jobs.txt:123]:
```suggestion
		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit." He then gives you <payment>.`
```

Comment (warp-core) [inline data/hai/unfettered jobs.txt:123]:
```suggestion
		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit." He then gives you <payment>.`
```

File: data/hai/unfettered jobs.txt [hunk 9]
```diff
@@ -108,3 +110,174 @@ mission "Unfettered Jump Drive 4"
...
+	deadline 12 2
+	cargo "medicals" 35 2 .05
+	source
+		attributes "unfettered"
+	stopover
+		distance 5 7
+		government "Hai"
+		attributes "spaceport"
+	destination
+		attributes "unfettered"
```
Comment (Hurleveur) [inline data/hai/unfettered jobs.txt:274]: alr
```suggestion
```

File: data/hai/unfettered jobs.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt:132]: I hid the big one and made the small one more unlikely, should be good now I think

File: data/hai/unfettered jobs.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt]: did it that way now, I'll do all it takes for it to be merged x)

File: data/hai/unfettered jobs.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/unfettered jobs.txt]: Nvm I was wrong on that, changed it hope its good now

**Comment by Hurleveur** [comment]
@Anarchist2 I think that I addressed all review comments, just took a look back on it and adjust the deadline/cost.

File: data/hai/unfettered jobs.txt [hunk 10]
```diff
@@ -108,3 +110,211 @@ mission "Unfettered Jump Drive 4"
...
+	description "Mine and  collect 35 tons of lead, then bring the minerals to <destination> for <payment>. Lead is available in the asteroid field in Wah Yoot."
+	source
+		attributes "unfettered"
+		not planet "Warfeed"
+	destination "Darkcloak"
+	to offer
+		random < 40
+		"cargo space" > 30
+	on complete
+		payment 200000
```
Comment (Amazinite) [inline data/hai/unfettered jobs.txt:239]:
```suggestion
		payment 95000
```

File: data/hai/unfettered jobs.txt [hunk 11]
```diff
@@ -108,3 +110,211 @@ mission "Unfettered Jump Drive 4"
...
+	description "Mine and collect 70 tons of lead, then bring the minerals to <destination> for <payment>. Lead is available in the asteroid field in Wah Yoot."
+	source
+		attributes "unfettered"
+		not planet "Warfeed"
+	destination "Darkcloak"
+	to offer
+		random < 10
+		"cargo space" > 55
+	on complete
+		payment 450000
```
Comment (Amazinite) [inline data/hai/unfettered jobs.txt:256]:
```suggestion
		payment 210000
```

File: data/hai/unfettered jobs.txt [hunk 12]
```diff
@@ -108,3 +110,215 @@ mission "Unfettered Jump Drive 4"
...
+		"The"
+	word
+		" Unfettered unloading the materials"
+	word
+		" you mined"
+		" you gathered"
+		" you brought"
+	word
+		" seem "
+		" appear"
```
Comment (tibetiroka) [inline data/hai/unfettered jobs.txt:127]:
```suggestion
		" appear "
```

File: data/hai/unfettered jobs.txt [hunk 13]
```diff
@@ -108,3 +110,215 @@ mission "Unfettered Jump Drive 4"
...
+	word
+		"satisfied."
+		"enthused."
+		"content."
+		"happy."
+	word
+		`"Keep this up and we will be building new ships in no time!`
```
Comment (tibetiroka) [inline data/hai/unfettered jobs.txt:134]:
```suggestion
		` "Keep this up and we will be building new ships in no time!`
```

File: data/hai/unfettered jobs.txt [hunk 14]
```diff
@@ -108,3 +110,215 @@ mission "Unfettered Jump Drive 4"
...
+		`"This will allow us to repair our ships!`
```
Comment (tibetiroka) [inline data/hai/unfettered jobs.txt:135]:
```suggestion
		` "This will allow us to repair our ships!`
```

File: data/hai/unfettered jobs.txt [hunk 15]
```diff
@@ -108,3 +110,215 @@ mission "Unfettered Jump Drive 4"
...
+		"Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere."
+	word
+		` "`
+	word
+		`This will allow us to repair our houses, our hospitals, even our schools. For that you have my deepest thanks.`
+		`Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit.`
+	word
+		`" `
+	word
+		"They then gives you <payment>."
```
Comment (tibetiroka) [inline data/hai/unfettered jobs.txt:165]:
```suggestion
		"They then give you <payment>."
```

**Review by tibetiroka — CHANGES_REQUESTED** [review]

**Review by tibetiroka — APPROVED** [review]

**Pull request merged**